--- origsrc/smplayer-0.6.2/Makefile	2008-04-28 19:47:10.000000000 -0500
+++ src/smplayer-0.6.2/Makefile	2008-09-07 02:21:44.121912000 -0500
@@ -1,13 +1,13 @@
 
-PREFIX=/usr/local
+PREFIX=/usr
 #PREFIX=/tmp/smplayer
 
 CONF_PREFIX=$(PREFIX)
 
 DATA_PATH=$(PREFIX)/share/smplayer
-DOC_PATH=$(PREFIX)/share/doc/packages/smplayer
+DOC_PATH=$(PREFIX)/share/doc/smplayer
 TRANSLATION_PATH=$(PREFIX)/share/smplayer/translations
-CONF_PATH=$(CONF_PREFIX)/etc/smplayer
+CONF_PATH=/etc/smplayer
 THEMES_PATH=$(PREFIX)/share/smplayer/themes
 SHORTCUTS_PATH=$(PREFIX)/share/smplayer/shortcuts
 
