--- origsrc/smplayer-0.6.5.1/Makefile	2008-11-01 18:32:22.000000000 -0500
+++ src/smplayer-0.6.5.1/Makefile	2008-11-23 20:48:13.753331800 -0600
@@ -1,11 +1,11 @@
 
-PREFIX=/usr/local
+PREFIX=/usr
 #PREFIX=/tmp/smplayer
 
 CONF_PREFIX=$(PREFIX)
 
 DATA_PATH=$(PREFIX)/share/smplayer
-DOC_PATH=$(PREFIX)/share/doc/packages/smplayer
+DOC_PATH=$(PREFIX)/share/doc/smplayer
 TRANSLATION_PATH=$(PREFIX)/share/smplayer/translations
 THEMES_PATH=$(PREFIX)/share/smplayer/themes
 SHORTCUTS_PATH=$(PREFIX)/share/smplayer/shortcuts
@@ -59,7 +59,7 @@ install: src/smplayer
 	install -m 644 Changelog *.txt $(DESTDIR)$(DOC_PATH)
 
 	-install -d $(DESTDIR)$(DOC_PATH)
-	tar -C docs/ --exclude=.svn -c -f - . | tar -C $(DESTDIR)$(DOC_PATH) -x -f -
+	tar -C docs/ --exclude=.svn -h -c -f - . | tar -C $(DESTDIR)$(DOC_PATH) -x -f -
 
 	-install -d $(DESTDIR)$(SHORTCUTS_PATH)
 	cp src/shortcuts/* $(DESTDIR)$(SHORTCUTS_PATH)
