--- origsrc/smplayer-0.6.5.1/Makefile	2008-11-01 18:32:22.000000000 -0500
+++ src/smplayer-0.6.5.1/Makefile	2008-11-23 20:48:13.753331800 -0600
@@ -1,11 +1,11 @@
 
-PREFIX=/usr/local
+PREFIX=/usr
 #PREFIX=/tmp/smplayer
 
 CONF_PREFIX=$(PREFIX)
 
 DATA_PATH=$(PREFIX)/share/smplayer
-DOC_PATH=$(PREFIX)/share/doc/packages/smplayer
+DOC_PATH=$(PREFIX)/share/doc/smplayer
 TRANSLATION_PATH=$(PREFIX)/share/smplayer/translations
 THEMES_PATH=$(PREFIX)/share/smplayer/themes
 SHORTCUTS_PATH=$(PREFIX)/share/smplayer/shortcuts
