--- origsrc/smplayer-17.4.0/Makefile	2017-01-29 18:25:26.000000000 -0600
+++ src/smplayer-17.4.0/Makefile	2017-04-02 22:00:49.074560100 -0500
@@ -1,11 +1,11 @@
 
-PREFIX=/usr/local
+PREFIX=/usr
 #PREFIX=/tmp/smplayer
 
 CONF_PREFIX=$(PREFIX)
 
 DATA_PATH=$(PREFIX)/share/smplayer
-DOC_PATH=$(PREFIX)/share/doc/packages/smplayer
+DOC_PATH=$(PREFIX)/share/doc/smplayer
 TRANSLATION_PATH=$(PREFIX)/share/smplayer/translations
 THEMES_PATH=$(PREFIX)/share/smplayer/themes
 SHORTCUTS_PATH=$(PREFIX)/share/smplayer/shortcuts
@@ -49,7 +52,7 @@ install: all
 	install -m 755 webserver/simple_web_server $(DESTDIR)$(PREFIX)/bin/
 
 	-install -d $(DESTDIR)$(DOC_PATH)
-	tar -C docs/ --exclude=.svn -c -f - . | tar -C $(DESTDIR)$(DOC_PATH) -x -f -
+	tar -C docs/ --exclude=.svn -h -c -f - . | tar -C $(DESTDIR)$(DOC_PATH) -x -f -
 
 	-install -d $(DESTDIR)$(SHORTCUTS_PATH)
 	cp src/shortcuts/* $(DESTDIR)$(SHORTCUTS_PATH)
